fal/controllers/anime_stats.py

```diff
@@ -156,6 +214,14 @@ def populate_anime_weekly_stats() -> None:
     year = config.getint("season info", "year")
     week = config.getint("weekly info", "current-week")
 
+    if is_week_to_calculate("scoring.simulcast", week) and simulcast_lines is None:
+        raise ValueError(f"simulcast file is required for week {week}")
+    if is_week_to_calculate("scoring.license", week) and licenses_lines is None:
```

Comment:
Good check here.

Do u think there's any need to store the licensing and simulcasts in the db? I didn't add them to the tables but I realize we do store everything else like aces, swaps, and wildcards. 

That being said, unlike everything else, license and simulcasts are a one time thing that doesn't affect any other part of the season...

Reply:
Another option is we call this phase one and leave that as a lower priority issue to tackle in the future.

Reply:
Now that I think about it, we might need to store it in the database because the weekly details post has information about whether the anime was licensed/simulcasted. However, we could leave this as phase 1 and have the code to generate the weekly details post just read from the text files and call the methods from `anime_stats.py` to get the anime that are licensed or simulcasted.